docs/README.md

```diff
@@ -156,6 +160,14 @@ result = lambda_handler(event=valid_input_event)
 assert result == {"body": '{"foo": [1.0, 2.2, 3.0]}', "statusCode": 200, "headers":{}}
 ```
 
+### arrays without specification in json schema
+
+If you supply an array, (comma seperated value),  to a query arg and you *dont* include a specification for the field in your jsonschema *then*: 
+
+1. the value will be unpacked as a string array
+2. all values will be tried cast to floats
```

Comment:
It's not clear if this means all fields must cast successfully or not, eg. "a,14,4,b" will be `'a', 14, 4, 'b'` or `'a','14','4','b'`

Reply:
Also does this mean you will get different types to the handler depending on what args are passed?